env/sugarizer.ini

```diff
@@ -1,57 +1,55 @@
-version=1.4.0
```

Comment:
Make as few changes in `.ini` file as you can.

Make sure to avoid changing the order of `favorites` (it's changed in this PR).